lib/matplotlib/tests/test_quiver.py

```diff
@@ -133,6 +134,20 @@ def test_barbs():
              cmap='viridis')
 
 
+@cleanup
+@raises(ValueError)
```

Comment:
I'm not seeing anywhere that we're explicitly using pytest yet...I sense this shouldn't be the first. 😞